packages/smooth_app/lib/data_models/user_preferences.dart

```diff
@@ -141,4 +144,11 @@ class UserPreferences extends ChangeNotifier {
 
   String? getDevModeString(final String tag) =>
       _sharedPreferences.getString(tag);
+
+  Future<void> setActiveAttributeGroup(final String value) async =>
+      _sharedPreferences.setString(_TAG_ACTIVE_ATTRIBUTE_GROUP, value);
+
+  String get activeAttributeGroup =>
+      _sharedPreferences.getString(_TAG_ACTIVE_ATTRIBUTE_GROUP) ??
+      'nutritional_quality'; // TODO(monsieurtanuki): relatively safe but not nice to put a hard-coded value (even when highly probable)
```

Comment:
Not possible to just open the first one in the list ?

Reply:
Yes we could, but it would mean first parsing all the groups and checking if nobody is expanded. I was a bit blocked by the fact that at some point getting all the groups was `async` and I needed that information when building a `Widget` (therefore not `async`). Does not seem impossible to code, but at the time that didn't look like a priority (hence the `TODO`).
Tell me if you insist.